WinUtils.cpp

```diff
@@ -104,37 +78,23 @@ void GetAllSubfolders(const std::string& path, std::set<std::string>& subfolders
 
 void GetAllFilesInFolderRecursive(const std::string& path, std::set<std::string>& filenames)
 {
-	struct _finddata_t provinceFileData;
-	intptr_t	fileListing	= NULL;
-	std::list<std::string> directories;
-	directories.push_back("");
-
-	while (!directories.empty())
+	for (auto& p : fs::recursive_directory_iterator(fs::u8path(path)))
 	{
-		if ((fileListing = _findfirst((path + directories.front() + "/*").c_str(), &provinceFileData)) == -1L)
+		if (!p.is_directory())
 		{
-			LOG(LogLevel::Error) << "Could not open directory " << path << '/' << directories.front() << "/*";
-			exit(-1);
+#if __linux__
```

Comment:
This is the Windows file, specifically excluded from Linux builds. You can drop this.